usecases/monitoring/prometheus.go

```diff
@@ -714,6 +720,29 @@ func newPrometheusMetrics() *PrometheusMetrics {
 			Help:    "Number of requests required to process an entire (user) batch",
 			Buckets: []float64{1, 2, 5, 10, 100, 1000},
 		}, []string{"vectorizer"}),
+		TokenizerDuration: promauto.NewHistogramVec(prometheus.HistogramOpts{
+			Name:    "tokenizer_duration_microseconds",
```

Comment:
Can we keep this in seconds? and adjust the bucket size to capture lower resolution.

Rationale is to keep all the metrics in seconds (standadize). Couldn't over state it's purpose when using promql joins to compare different metrics.

[Prometheus recommends it](https://prometheus.io/docs/practices/naming/#base-units)

[Even Otel recommends it.](https://github.com/open-telemetry/opentelemetry-specification/issues/2977#issue-1460359959)